generators/pkg/manager/manage.go

```diff
@@ -0,0 +1,76 @@
+package manager
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+// Create the Kubernetes client
+func GetKubeClient(kubeconfig *string) (client.Client, error) {
```

Comment:
I don't think there's any reason this needs to be a `*string`. Let's use `string` instead.